app/src/main/java/com/sun_asterisk/moviedb_50/screen/listmovie/ListMovieContract.kt

```diff
@@ -0,0 +1,34 @@
+package com.sun_asterisk.moviedb_50.screen.listmovie
+
+import com.sun_asterisk.moviedb_50.data.model.Movie
+import com.sun_asterisk.moviedb_50.screen.base.BasePresenter
+import com.sun_asterisk.moviedb_50.utils.Constant
+
+interface ListMovieContract {
+    /**
+     * View
+     */
+    interface View {
+        fun onGetMoviesByGenresIDSuccess(movies: List<Movie>)
+        fun onGetMoviesByCastIDSuccess(movies: List<Movie>)
+        fun onGetMoviesByProduceIDSuccess(movies: List<Movie>)
+        fun onGetMoviesPopularSuccess(movies: List<Movie>)
+        fun onGetMoviesTopRateSuccess(movies: List<Movie>)
+        fun onGetMoviesNowPlayingSuccess(movies: List<Movie>)
+        fun onGetMoviesUpcomingSuccess(movies: List<Movie>)
+        fun onGetMoviesByMovieNameSuccess(movies: List<Movie>)
```

Comment:
chỉ cần 1 func  + thêm 1 param tag để phân biệt loại là được rồi.
Nó lặp lại hoàn toàn mà.